x-pack/plugins/monitoring/public/components/elasticsearch/shard_allocation/components/shard.js

```diff
@@ -80,23 +80,24 @@ export class Shard extends React.Component {
 
   render() {
     const shard = this.props.shard;
-    let tooltip;
-    if (this.state.tooltipVisible) {
-      tooltip = (
-        <div
-          className="shard-tooltip"
-          data-test-subj="shardTooltip"
-          data-tooltip-content={this.props.shard.tooltip_message}
-        >
-          {this.props.shard.tooltip_message}
-        </div>
-      );
-    }
-
     const classes = calculateClass(shard);
     const color = getColor(classes);
     const classification = classes + ' ' + shard.shard;
 
+    let shardUi = (
+      <EuiTextColor color={color}>
```

Comment:
Is this `color` code working? In the screenshot, the color doesn't match any of the badge colors. Perhaps the `shardUi` should be a badge instead of a dark block of text with a color that may not show well on that background?